source/extensions/tracers/opentelemetry/tracer.cc

```diff
@@ -120,7 +121,24 @@ void Span::setAttribute(absl::string_view name, const OTelAttribute& attribute_v
   *span_.add_attributes() = key_value;
 }
 
-void Span::setTag(absl::string_view name, absl::string_view value) { setAttribute(name, value); }
+void Span::setTag(absl::string_view name, absl::string_view value) {
+  if (name == Tracing::Tags::get().HttpStatusCode) {
+    uint64_t status_code;
+    // For HTTP status codes in the 5xx range, as well as any other code the client failed to
```

Comment:
Can you check the formatting here? The comment looks like it has strange line breaks.

Reply:
reformated by `tools/local_fix_format.sh`